tests/h/search/index_test.py

```diff
@@ -310,6 +311,66 @@ def test_annotation_is_marked_deleted(self, es_client, factories, index, search)
         assert result.get('_source').get('deleted') is True
 
 
+class TestBatchIndexer(object):
+    def test_it_indexes_all_annotations(self, db_session, es_client, factories, pyramid_request):
+        annotations = factories.Annotation.create_batch(3)
+        ids = [a.id for a in annotations]
+
+        h.search.index.BatchIndexer(db_session, es_client, pyramid_request, es_client.index).index()
```

Comment:
The `BatchIndexer` construction is repeated with the same arguments in several test cases. I would suggest moving into a fixture (say `batch_indexer` that returns an instance of `BatchIndexer`).